app/controllers/sessions_controller.rb

```diff
@@ -1,7 +1,9 @@
 # frozen_string_literal: true
 
 class SessionsController < ApplicationController
-  skip_before_action :set_current_user
+  skip_before_action :set_current_user,
+                     :check_for_incomplete_incidents,
+                     :check_for_unclaimed_incidents
```

Comment:
Now that we check things involving the current user in these before actions, we can't do that in controller actions where there might not be a current user.

That said, in those instances we don't show the navbar buttons anyway, so this doesn't cause any changes in behavior.